src/Validation/Validator.php

```diff
@@ -211,6 +219,43 @@ public function getProvider($name)
     }
 
     /**
+     * Returns the default provider stored under that name if it exists.
+     *
+     * @param string $name The name under which the provider should be retrieved.
+     * @return object|string|null
```

Comment:
With three different potential return value types, it might be worth describing which comes in which case.